lib/govuk_design_system_formbuilder/elements/date.rb

```diff
@@ -105,7 +122,8 @@ def input(segment, link_errors, width, value)
           pattern: pattern(segment),
           inputmode: 'numeric',
           value: value,
-          autocomplete: date_of_birth_autocomplete_value(segment)
+          autocomplete: date_of_birth_autocomplete_value(segment),
+          maxlength: maxlength,
```

Comment:
As we're already passing in `width` here for setting the input width I think we can just use it directly. Doing something like this will allow you to avoid having the `if`/`else` statements in `#day`, `#month` and `#year`.

```suggestion
          maxlength: (width if maxlength_enabled?)
```

Reply:
This is much cleaner :pray: